dynamic-plugins/imports/package.json

```diff
@@ -12,26 +12,26 @@
     "lint": "backstage-cli package lint"
   },
   "devDependencies": {
-    "@backstage/cli": "0.26.2",
+    "@backstage/cli": "0.26.4",
     "tar": "6.2.1"
   },
   "peerDependencies": {
-    "@janus-idp/backstage-plugin-3scale-backend": "1.4.12",
-    "@janus-idp/backstage-plugin-aap-backend": "1.5.10",
-    "@janus-idp/backstage-plugin-acr": "1.2.35",
-    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.2.16",
-    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.2.35",
-    "@janus-idp/backstage-plugin-keycloak-backend": "1.8.11",
-    "@janus-idp/backstage-plugin-nexus-repository-manager": "1.4.36",
-    "@janus-idp/backstage-plugin-ocm": "3.7.10",
-    "@janus-idp/backstage-plugin-ocm-backend": "3.5.12",
-    "@janus-idp/backstage-plugin-quay": "1.5.17",
-    "@janus-idp/backstage-plugin-rbac": "1.17.4",
-    "@janus-idp/backstage-plugin-tekton": "3.5.18",
-    "@janus-idp/backstage-plugin-topology": "1.18.14",
-    "@janus-idp/backstage-scaffolder-backend-module-quay": "1.3.10",
-    "@janus-idp/backstage-scaffolder-backend-module-regex": "1.3.10",
-    "@janus-idp/backstage-scaffolder-backend-module-servicenow": "1.3.11",
-    "@janus-idp/backstage-scaffolder-backend-module-sonarqube": "1.3.10"
+    "@janus-idp/backstage-plugin-3scale-backend": "1.5.6",
+    "@janus-idp/backstage-plugin-aap-backend": "1.6.6",
+    "@janus-idp/backstage-plugin-acr": "1.3.6",
+    "@janus-idp/backstage-plugin-analytics-provider-segment": "1.3.6",
+    "@janus-idp/backstage-plugin-jfrog-artifactory": "1.3.6",
+    "@janus-idp/backstage-plugin-keycloak-backend": "1.9.6",
+    "@janus-idp/backstage-plugin-nexus-repository-manager": "1.5.6",
+    "@janus-idp/backstage-plugin-ocm": "4.0.2",
+    "@janus-idp/backstage-plugin-ocm-backend": "4.0.2",
+    "@janus-idp/backstage-plugin-quay": "1.6.7",
+    "@janus-idp/backstage-plugin-rbac": "1.18.2",
```

Comment:
should we have rbac-node, rbac-common, and rbac-backend here too?

Reply:
Those are already statically installed in the showcase.
https://github.com/janus-idp/backstage-showcase/blob/3c9376cc455d6e014b1337f7ca1f9407f92ef371/packages/backend/package.json#L41-L42

Reply:
no, it doesn't need to be dynamic.